service/static/js/rest_api.js

```diff
@@ -271,4 +271,34 @@ $(function () {
 
     });
 
+    // ****************************************
+    // Toggle the status of a recommendation
+    // ****************************************
+
+    $("#toggle-btn").click(function () {
+
+        var product_id = $("#recommendation_product_id").val();
+        var related_product_id = $("#recommendation_related_product_id").val();
+
+        if (product_id && related_product_id){
+            var ajax = $.ajax({
+                type: "PUT",
+                url: "/recommendations/" + product_id + "/" + related_product_id + "/toggle"
+            })
+
+            ajax.done(function(res){
+                flash_message("Success");
+                $("#search-btn").trigger('click');
```

Comment:
Hi @lj894, you could just do 'update_form_data(res)', I think triggering the search button will cause a delay as a new request has to be sent and received.

Reply:
Ok. Because the toggle only returns the status, not the other attributes. so I thought it'd be safe to do another GET to make sure we get all the information from API so the toggle won't accidently update anything else. But I guess since we have tested the API elsewhere we can safe to assume the API works as we expected.

Reply:
um okay, I thought we agreed on changing the toggle endpoint to return the entire recommendation, not just the status, is there any reason to return only the status?